packages/server/postgres/migrations/1658834062488_createMeetingSeries.ts

```diff
@@ -0,0 +1,33 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "MeetingSeries" (
+      "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+      "meetingType" "MeetingTypeEnum" NOT NULL,
+      "title" TEXT NOT NULL,
+      "recurrenceRule" TEXT NOT NULL,
+      "duration" INT NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "cancelledAt" TIMESTAMP WITH TIME ZONE
```

Comment:
a little comment just to make sure we're on the same page. `TIMESTAMP WITH TIME ZONE` type in pg is stored as UTC internally but converted to time zone specified in pg config when returning the date. 

from pg docs 

> All timezone-aware dates and times are stored internally in UTC. They are converted to local time in the zone specified by the [TimeZone](https://www.postgresql.org/docs/current/runtime-config-client.html#GUC-TIMEZONE) configuration parameter before being displayed to the client.

imo it's best to have postgres set to UTC, too. this will guarantee we won't make any mistakes when calculating if the meeting should actually be closed, etc. 

to check what timezone is set you can run `SHOW TIMEZONE;`; to change `SET TIMEZONE='UTC` (it's db wise so we have to be very careful to not break anything else, if we need to actually do this)

Reply:
This probably won't be used when calculating recurrence effects - if this timestamp is used, it will probably only be used in the UI.